spec/graphql/mutations/password_resets/reset_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::PasswordResets::Reset, type: :graphql do
+  let(:membership) { create(:membership, user: create(:user, password: 'HelloLago!1')) }
+  let(:password_reset) { create(:password_reset, user: membership.user) }
+
+  let(:mutation) do
+    <<~GQL
+      mutation($input: ResetPasswordInput!) {
+        resetPassword(input: $input) {
+          token
+        }
+      }
+    GQL
+  end
+
+  it 'creates an invite for a new user' do
```

Comment:
title is incorrect, probably leftover:)

Reply:
Good catch ! I did moved the query inside the service and added tests for the expirtation 🙏